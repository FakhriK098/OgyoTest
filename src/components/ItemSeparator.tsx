import { StyleSheet, View } from 'react-native';
import React from 'react';

const ItemSeparator = () => {
  return <View style={styles.separator} />;
};

const styles = StyleSheet.create({
  separator: {
    height: 8,
  },
});

export default ItemSeparator;
