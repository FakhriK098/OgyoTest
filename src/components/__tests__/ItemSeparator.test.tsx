import React from 'react';
import { render } from '@testing-library/react-native';
import ItemSeparator from '../ItemSeparator';

describe('ItemSeparator', () => {
  it('should match snapshot', () => {
    const { toJSON } = render(<ItemSeparator />);
    expect(toJSON()).toMatchSnapshot();
  });
});
