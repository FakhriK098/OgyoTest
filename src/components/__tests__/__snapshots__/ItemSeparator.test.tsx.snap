// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemSeparator should match snapshot 1`] = `
<View
  style={
    {
      "height": 8,
    }
  }
/>
`;
